curriculum_algorithm/models.py

```diff
@@ -0,0 +1,122 @@
+from django.db import models
+
+POSITION_CHOICES = [
+    ('0', 'Any'),
+    ('1', 'Fall'),
+    ('2', 'Spring'),
+]
+
+
+class Course(models.Model):
+    course_number = models.CharField(max_length=8, unique=True)
```

Comment:
This field is the course number or the complete course id? I suggest changing the field name to avoid confusion